Clarify meeds association vs meeds dao (#289)
Meeds-io/whitepaper: en/3-meeds.md

# 3. MEED token – the currency of the WoM

> Its currency powers the WoM – the Meeds (MEED) token. Users are rewarded with MEED tokens (through engagement programs), purchase in stores, get Deeds, get financial returns on investments, and more.

Reviewer: ```suggestion
Its currency powers the WoM – the MEED token. Users are rewarded with MEED tokens through engagement programs, purchase in stores, get Deeds, get financial returns on investments, and more.
```

Author: applied.

> Its currency powers the WoM – the Meeds (MEED) token. Users are rewarded with MEED tokens (through engagement programs), purchase in stores, get Deeds, get financial returns on investments, and more.

Reviewer 2: ```suggestion
Its currency powers the World of Meed –  MEED token. Users are rewarded with MEED tokens (through engagement programs), purchase in stores, get Deeds, get financial returns on investments, and more.
```

> Its currency powers the WoM – the Meeds (MEED) token. Users are rewarded with MEED tokens (through engagement programs), purchase in stores, get Deeds, get financial returns on investments, and more.

Reviewer 3: ```suggestion
Its currency powers the WoM – the MEED token. Users are rewarded with MEED tokens through engagement programs, purchase in stores, get Deeds, get financial returns on investments, and more.
```

## 3.1. Tokenomics

The **maximum supply of MEED is limited to 100 million tokens**, minted progressively. 

> The Meeds Association has minted 5 Million MEED (5% of the total supply) at the start to acquire software IP from the funding members, pay for token emission legal fees, and hosting services. 

Reviewer 4: Meeds is used mainly to describe the organization while MEED is used for the token.

Author: Yes. In this sentence, the Meeds Association term was used on purpose to clarify who minted the first tokens.

The minting rate is a parameter of the Token Factory, set at *10 MEED per minute* at the start. A vote of the DAO may change this rate in the future. 
At such rate, the Token Factory mints around 5 Million MEED yearly (10*60*24*365); thus, the minting horizon is nearly 20 years.

## 3.2. Minting allocation

The Token Factory also rules the allocation of minted MEED tokens to  three categories of stakeholders :

- **Builders**: Contributors of the Meeds DAO that help build and run the WoM
- **Users**: Members in all communities of the WoM earn MEED as rewards for their contributions
- **Investors**: The Meeds DAO incentivizes MEED token owners to stake MEED or provide liquidity for the Meeds economy 

Each category of stakeholders receives a share of the MEED supply, which is subject to votes by the DAO.

## 3.3. Users – minting through engagement

Part of the continuously minted Meeds is allocated to sponsor user engagement through the Mintium model (see 2.6). A global engagement index is calculated periodically across all Hubs of the WoM to allocate MEED rewards to the Hubs. Mintium rewards are claimable by Hub leaders and Deed owners.

> ## 3.4. Builders – Earn for building the WoM

Reviewer 4: I seem to be a little confused. Comparing and contrasting the two statements "Meeds association DAO and community of Builders" and "Earn for building the WoM" do not relate to the same information. 
While the first talks about the Builders of Meeds DAO and community broadly;, the latter narrows down only to the benefits.

Author: agreed.  I'm going to simplify like this : 
3.3. Users
3.4. Builders
3.3. Investors

Also, there are plans to introduce bonds and move towards protocol-owned liquidity progressively. The latter would contribute to the project's long-term sustainability - as mentioned above, transaction fees will fund the operating expenses.

Additionally, on behalf of the Meeds DAO, the Meeds Association is entitled to facilitate OTC (off-the-shelf) purchases of more extensive amounts of MEED tokens.

**Staking / long-term investment**
Meeds DAO encourages investors to invest long-term by staking their MEED by providing yields on staked MEEDs in proportion to the xMEED holdings. Yield profits can only be realized by unstaking MEED.

xMEED is the token used for voting on DAO proposals.

Additionally, xMEED holders earn minting points that entitle them to mint Deed NFTs. This mechanism increases the demand for MEED tokens. Minting points accrue over time as long as the stakeholder maintains his stake. The formula below allows you to calculate minting points :

 $$ P = xMeeds / (xMeeds + 12000) * T / 240 $$

>  Where:

Reviewer 4: The space should be deleted here

## 3.6. Meeds DAO governance

> xMEED token holders can freely participate in the governance of the Meeds DAO by voting with their tokens on decisions such as managing the budget allocation or electing DAO board members.

Author: xMEED is the token you get when you stake MEED. it's a different token and this is explained in the previous paragraph. So there is no error here.

## 3.7. Decentralized finance (DeFi) user services

The WoM will introduce the blockchain world to many people who do not use it today.

> Indeed, Meeds can gamify and set a monetary value on any work (whether digital or not) and reward it with MEED tokens. Thanks to an easy-to-use, blockchain-based wallet app that simplifies access to the crypto world for an average user, rewards are tangible.

Reviewer 5: I think that the use of the word "contribution" instead of "work" makes it more user-friendly and "Thanks" is an informal team not suitable in an official document.

Beyond the wallet and a store where they can purchase perks for Meeds, we plan to introduce every user to several end-user products powered by the Decentralized Finance protocols, such as:

- The ability to stake MEED tokens and get yields (generate revenues)

> - The ability to use MEED tokens as collateral to borrow stablecoins such as USDC (leveraging first a [Fuse pool](https://app.rari.capital/fuse) then registering the Meeds token to [AAVE](https://aave.com/).

Reviewer 4: ``` suggestion
- The ability to use MEED tokens as collateral to borrow stablecoins such as USDC (leveraging first a [Fuse pool](https://app.rari.capital/fuse) then registering the Meeds token to [AAVE](https://aave.com/).
```
